src/meta-srv/src/service/admin.rs

```diff
@@ -257,6 +259,17 @@ pub fn admin_axum_router(metasrv: Arc<Metasrv>) -> AxumRouter {
     let recovery_handler = Arc::new(RecoveryHandler {
         manager: metasrv.runtime_switch_manager().clone(),
     });
+    let table_id_sequence_handler = Arc::new(TableIdSequenceHandler {
+        table_id_sequence: metasrv.table_id_sequence().clone(),
+        runtime_switch_manager: metasrv.runtime_switch_manager().clone(),
+    });
+    let sequence_router = AxumRouter::new().nest(
+        "/table",
+        AxumRouter::new()
+            .route("/next-id", routing::get(sequencer::get_next_table_id))
+            .route("/next-id", routing::post(sequencer::set_next_table_id))
```

Comment:
better make it explicit:

```suggestion
            .route("/set-next-id", routing::post(sequencer::set_next_table_id))
```